core/assets/bundles/bundle.properties

```diff
@@ -673,4 +669,113 @@ tutorial.router = Factories need resources to function.\nCreate a router to spli
+block.omega-mech-pad.description = Leave your current vessel and change into a bulky and well-armored mech, made for front-line assaults.\nUse the pad by double tapping while standing on it.
+block.spirit-factory.description = Produces light drones which mine ore and repair blocks.
+block.phantom-factory.description = Produces advanced drone units which are significantly more effective than a spirit drone.
+block.wraith-factory.description = Produces fast, hit-and-run interceptor units.
+block.ghoul-factory.description = Produces heavy carpet bombers.
+block.dagger-factory.description = Produces basic ground units.
+block.titan-factory.description = Produces advanced, armored ground units.
+block.fortress-factory.description = Produces heavy artillery ground units.
+block.revenant-factory.description = Produces heavy laser ground units.
+block.repair-point.description = Periodically repairs units in its vicinity.
+block.command-center.description = Allows changing friendly AI behavior. Currently, attack, retreat and patrol commands are supported.
+block.conduit.description = Basic liquid transport block. Works like a conveyor, but with liquids. Best used with extractors, pumps or other conduits.
+block.pulse-conduit.description = Advanced liquid transport block. Transports liquids faster and stores more than standard conduits.
+block.phase-conduit.description = Advanced liquid transport block. Uses power to teleport liquids to a connected phase conduit over several tiles.
+block.liquid-router.description = Accepts liquids from one direction and outputs them to up to 3 other directions equally. Can also store a certain amount of liquid. Useful for splitting the liquids from one source to multiple targets.
+block.liquid-tank.description = Stores a large amount of liquids. Use it for creating buffers when there is a non-constant demand of materials or as a safeguard for cooling vital blocks.
+block.liquid-junction.description = Acts as a bridge for two crossing conduits. Useful in situations with two different conduits carrying different liquids to different locations.
+block.bridge-conduit.description = Advanced liquid transport block. Allows transporting liquids over up to 3 tiles of any terrain or building.
+block.mechanical-pump.description = A cheap pump with slow output, but no power consumption.
+block.rotary-pump.description = An advanced pump which doubles up speed by using power.
+block.thermal-pump.description = The ultimate pump. Three times as fast as a mechanical pump and the only pump which is able to retrieve lava.
+liquid.water.description = A large part of your body consists of it. Additionally, it speeds up or cools down many things.
+liquid.lava.description = Can be transformed into[LIGHT_GRAY] stone[], used for generating power or used as ammo for certain turrets.
+liquid.oil.description = Can be burnt, exploded or used as a coolant.
+liquid.cryofluid.description = Most efficient liquid for cooling things down.
```

Comment:
Would be better phrased as "_The_ most efficient liquid for cooling things down."